README.md

```diff
@@ -13,7 +13,7 @@
 
 ## Live Demo
 
-[Live Demo Link](https://wizardly-borg-3c9b3f.netlify.app/)
+[Live Demo Link](https://suspicious-einstein-3746cf.netlify.app/)
```

Comment:
- [ ] OPTIONAL - In the readme, consider updating the app screenshot to match the present app look.

Reply:
Thanks for this remember me this